core/vm/eips.go

```diff
@@ -197,3 +201,133 @@ func enable3860(jt *JumpTable) {
 	jt[CREATE].dynamicGas = gasCreateEip3860
 	jt[CREATE2].dynamicGas = gasCreate2Eip3860
 }
+
+func enable3540(jt *JumpTable) {
+	// Do nothing.
+}
+
+func enable3670(jt *JumpTable) {
+	// Do nothing.
```

Comment:
Per EIP-3670 `CALLCODE` & `SELFDESTRUCT` should be marked as `legacyOnly`.

Reply:
Resolved by [6fbc90f](https://github.com/ledgerwatch/erigon/pull/6215/commits/6fbc90f51c33e283ada243f847e7f52870a31db5)